src/mmw/js/src/modeling/models.js

```diff
@@ -751,6 +758,7 @@ var ScenariosCollection = Backbone.Collection.extend({
             aoi_census: aoi_census
         });
 
+        scenario.waitToRefresh = true;
```

Comment:
Attributes on models should be set using `scenario.set('foo', 'bar').` I know this is sort of like a meta attribute, but I think we should still probably use the Backbone methods.